{"description": "An issue was discovered in Django 2.2 before 2.2.26, 3.2 before 3.2.11, and 4.0 before 4.0.1. `UserAttributeSimilarityValidator` incurred significant overhead in evaluating a submitted password that was artificially large in relation to the comparison values. In a situation where access to user registration was unrestricted, this provided a potential vector for a denial-of-service attack.", "methods": ["Used a denial-of-service attack vector through unrestricted access to user registration.", "Leveraged significant overhead in evaluating a submitted password to cause a denial-of-service."]}